docs/content/references/yaml-metadata.md

```diff
@@ -0,0 +1,181 @@
+---
+robots: index, follow
+lang: en
+dir: ltr
+breaks: true
+---
+
+# Supported YAML metadata
+
```

Comment:
Maybe add a short sentence like:

With a YAML frontmatter you can change certain metadata and aspects of your note.

Reply:
Yes, but please put the sentence below the heading.
So that it looks like this:

```markdown
# Supported YAML metadata

With a YAML frontmatter you can change certain metadata and aspects of your note.

First you need to insert syntax like this at the **start** of the note:
```